ExtractorLauncher/ExtractorStarter.cs

```diff
@@ -79,6 +79,16 @@ public static class ExtractorStarter
                 var parsed = CogniteTime.ParseTimestampString(config.History.EndTime);
                 if (parsed == null) return $"Invalid history end time: {config.History.EndTime}";
             }
+            if (config.Source.SamplingInterval != null)
+            {
+                log.LogWarning("source.sampling-interval is deprecated. Use subscriptions.sampling-interval instead.");
+                config.Subscriptions.SamplingInterval = config.Source.SamplingInterval.Value;
```

Comment:
What happens if the same property is somehow, also set in the `Subscriptions` config section? Which one takes precedence if it, hypothetically, happens?

Reply:
The source does, due to this. However it is hard to detect whether the value is set in Subscriptions, since it has a default value there.

This is fine. If somebody _does_ set it in `source`, it should mean that they are using old config, and that they meant to set it.

Reply:
Oh right! Makes sense now.